src/Symfony/Component/Validator/Resources/translations/validators.de.xlf

```diff
@@ -338,6 +338,30 @@
                 <source>This collection should contain only unique elements.</source>
                 <target>Diese Sammlung darf keine doppelten Elemente enthalten.</target>
             </trans-unit>
+            <trans-unit id="88">
+                <source>This value should be positive.</source>
+                <target>Dieser Wert sollte positiv sein.</target>
+            </trans-unit>
+            <trans-unit id="89">
+                <source>This value should be either positive or zero.</source>
+                <target>Dieser Wert sollte entweder positiv oder 0 sein.</target>
+            </trans-unit>
+            <trans-unit id="90">
+                <source>This value should be negative.</source>
+                <target>Dieser Wert sollte negativ sein.</target>
+            </trans-unit>
+            <trans-unit id="91">
+                <source>This value should be either negative or zero.</source>
+                <target>Dieser Wert sollte entweder negativ oder 0 sein.</target>
+            </trans-unit>
+            <trans-unit id="92">
+                <source>This value is not a valid timezone.</source>
+                <target>This value is not a valid timezone.</target>
+            </trans-unit>
+            <trans-unit id="93">
+                <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
+                <target>Dieses Passwort ist Teil eines Datenlecks, es darf nicht verwendet werden.</target>
```

Comment:
```suggestion
                <target>Dieses Passwort ist Teil eines Datenlecks, es darf nicht verwendet werden.</target>
                <target>Dieses Passwort ist Teil eines Datenlecks, es darf nicht verwendet werden. Bitte nutzen Sie ein anderes Passwort.</target>
```

Reply:
@derrabus, @dbrumann and me discussed this and we decided to leave this as is as it doesn't add any real value. Do you agree with that?

Reply:
it would still make sense as that'd be the full translation of the english version

Reply:
@xabbuh I'm fine with that. I was not really sure if we should either use "Bitte nutzen Sie ein anderes Passwort." or "Bitte nutze ein anderes Passwort." anyway. So removing it completely will not bring up this discussion :-)